src/ohow.ml

```diff
@@ -36,8 +36,20 @@ let build_page content =
     | Some s -> s
     | None -> ""
   in
+  let stylesheets = ["bootstrap_custom.css"; "style.css"; "syntax.css"] |> List.map (fun f -> "https://ocsigen.github.io/css/" ^ f) in
+  let scripts = ["https://cdnjs.cloudflare.com/ajax/libs/prism/1.9.0/components/prism-core.min.js";
+                 "https://cdnjs.cloudflare.com/ajax/libs/prism/1.9.0/components/prism-ocaml.min.js";
+                 "https://cdnjs.cloudflare.com/ajax/libs/prism/1.9.0/components/prism-clike.min.js";
+                 "https://cdnjs.cloudflare.com/ajax/libs/prism/1.9.0/components/prism-reason.min.js";
+                 "https://cdnjs.cloudflare.com/ajax/libs/prism/1.9.0/components/prism-javascript.min.js";
+                 "https://ocsigen.github.io/js/client.js";
+                 "https://ocsigen.github.io/js/viewport_ie10_hack.js"]
```

Comment:
Is it possible to have relative links for these two?

Reply:
Absolute urls are needed to have it working locally without making assumptions on the directories structure (option `--local` not implemented yet).